src/containers/treePage/index.tsx

```diff
@@ -1,93 +1,82 @@
-import { connect, useDispatch, useSelector } from 'react-redux';
-import { LIGHT_GREY, DARK_GREEN, TEXT_GREY } from '../../utils/colors';
-import { Gap } from '../../components/themedComponents';
-import styled from 'styled-components';
 import {
-  SiteReducerState,
-  ProtectedSitesReducerState,
+  ActivityRequest,
   Entry,
+  ProtectedSitesReducerState,
+  SiteReducerState,
+  TreeCare,
 } from './ducks/types';
+import { connect, useDispatch, useSelector } from 'react-redux';
+import { LIGHT_GREY, TEXT_GREY } from '../../utils/colors';
+import styled from 'styled-components';
 import {
-  mapStewardshipToTreeCare,
-  isTreeAdopted,
   getLatestEntry,
+  isTreeAdopted,
+  mapStewardshipToTreeCare,
 } from './ducks/selectors';
 import { asyncRequestIsComplete } from '../../utils/asyncRequest';
 import { C4CState } from '../../store';
 import { getAdoptedSites, getSiteData } from './ducks/thunks';
 import protectedApiClient from '../../api/protectedApiClient';
+import TreeBackground from '../../assets/images/grey-logo.png';
+import { RecordStewardshipRequest } from '../../components/forms/ducks/types';
+import useWindowDimensions, {
+  WindowTypes,
+} from '../../components/windowDimensions';
+import ReturnButton from '../../components/returnButton';
+import TreeInfo from '../../components/treeInfo';
+import TreeActivity from '../../components/treeActivity';
 
 const { Paragraph, Title } = Typography;
 
 const StyledCard = styled(Card)`
-  width: 250px;
   height: 100px;
+  line-height: 15px;
   border: solid 1px ${LIGHT_GREY};
-  margin: 7px;
+  overflow: auto;
 `;
 
 const TreePageContainer = styled.div`
-  width: 95vw;
+  width: 90vw;
   margin: 30px auto auto;
 `;
 
 const TreeMainContainer = styled.div`
   margin: 50px 30px 30px;
 `;
 
-const TreeInfoContainer = styled.div`
-  width: 90%;
-  margin: auto;
-`;
-
-const EntrySpace = styled.div`
-  display: flex;
-  flex-wrap: wrap;
+const MobileTreeMainContainer = styled.div`
+  margin: 50px 10px 30px;
 `;
 
-const TreeCareTitle = styled(Paragraph)`
-  margin: 15px 15px 50px;
-  font-size: 24px;
-  font-weight: bold;
-  color: ${DARK_GREEN};
+const TreeInfoContainer = styled.div`
+  height: 100%;
+  background: url(${TreeBackground}) no-repeat bottom right;
+  background-size: contain;
+  margin: auto;
 `;
 
 const TreeCareContainer = styled.div`
+  margin-top: 5vh;
   border: solid 1px ${LIGHT_GREY};
   max-height: 80%;
-  margin-top: 80px;
+  padding: 30px 15px 5px;
   overflow: auto;
 `;
 
-const CareEntry = styled.div`
-  margin: 15px;
-`;
-
-const StewardshipContainer = styled.div`
-  margin-top: 40px;
-`;
-
-const EntryDate = styled(Paragraph)<TitleProps>`
-  display: inline;
-  text-align: center;
-  line-height: 0px;
-  font-size: 18px;
-  font-weight: bold;
-  color: ${DARK_GREEN};
+const MobileTreeCareContainer = styled.div`
+  margin-top: 5vh;
+  border: solid 1px ${LIGHT_GREY};
+  max-height: 50vh;
+  padding: 30px 15px 5px;
+  overflow: auto;
```

Comment:
💭 I'm not sure how much sense it makes for this to scroll on mobile

Reply:
Maybe pagination? Or just restrict how many entries are shown?